### PR TITLE
fixes #8142 feat(nimbus): add new android 13 users targeting config and update test context

### LINKED ISSUE #8142: Add Advanced Filtering to only run for users on Android 13
Google first released Android 13 in August 2022.

This experiment needs to only go to users on the FIRST RUN AND Android 13 or higher (in case a dot release comes) https://experimenter.services.mozilla.com/nimbus/android-pre-permission-prompt-experiment/edit/audience

brief: https://docs.google.com/document/d/1yPqTWGtJ8DPrlUoVqtX8AOtpMCoE7WGdb7R4l9BqYT4/edit#

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/EXP-3059)

### PR DESCRIPTION
**Pull request by jeddai**
Because
    
- New targeting is needed for [this Android experiment](https://docs.google.com/document/d/1yPqTWGtJ8DPrlUoVqtX8AOtpMCoE7WGdb7R4l9BqYT4/edit#heading=h.j0lsegsodnsd)

This commit
   
- Adds the required targeting
- Adds a value to the test targeting context based on its [value in the Fenix repo](https://github.com/mozilla-mobile/fenix/blob/main/app/src/main/java/org/mozilla/fenix/gleanplumb/CustomAttributeProvider.kt#L36)

### CODE REVIEW

File: app/tests/integration/nimbus/test_mobile_targeting.py [hunk 1]
```diff
@@ -78,6 +78,7 @@ def test_check_mobile_targeting(
             "days_since_update": 1,
             "days_since_install": 1,
             "isFirstRun": "true",
+            "is_first_run": True,
```
Comment (jaredlockhart) [inline app/tests/integration/nimbus/test_mobile_targeting.py:81]: Why is there both `isFirstRun` and `is_first_run`?

Comment (jeddai) [reply app/tests/integration/nimbus/test_mobile_targeting.py:81]: Both exist within the Fenix repo, `isFirstRun` is a string and only there for legacy purposes, they've moved to `is_first_run` as a Boolean going forward.

Comment (jaredlockhart) [reply app/tests/integration/nimbus/test_mobile_targeting.py:81]: Ah interesting 👍

**Review by jaredlockhart — APPROVED** [review]
Looks good thank you 👍

**Pull request merged**